index.js

```diff
@@ -66,9 +74,48 @@ function clickHandle (event) {
         
         let element = document.getElementById('message');
         element.innerHTML = `<p>${data.overview}</p>`;
-        
-        console.log(data);
-        
     });
 }
 
+const renderTopMovies = function() {
+
+    let cardsList = document.getElementsByClassName('movieCard');
```

Comment:
```suggestion
    const cardsList = document.getElementsByClassName('movieCard');
```

Es `const` porque no estás reasignando la variable.